daily-challenges/2025-01/04/hunghoang3110.ts

```
@@ -0,0 +1,56 @@
+    name: string;
+    rank: string;
+    chips: number;
+};
+
+type Ranked = {
+    name: string;
+    score: number;
+};
+
+function calculatePokerScore(players: Player[]): { name: string, score: number }[] {
+    let arrayRanked: Ranked[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'arrayRanked' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)